packages/runtime/lib/management-api.js

```diff
@@ -221,7 +222,7 @@ async function startManagementApi (runtime, configManager) {
 
     managementApi.addHook('onClose', async () => {
       if (platform() !== 'win32') {
-        await rm(runtimePIDDir, { recursive: true, force: true }).catch()
+        await rm(runtimePIDDir, { recursive: true, force: true })
```

Comment:
I have a rough feeling those `catch()` are actually needed

Reply:
I don't see why they were here, but @ivan-tymoshenko might have more info

Reply:
Nevermind. It is actually needed to avoid failures. Taking care of these.

Reply:
I'm actually restructuring the whole code base where `await rm` was used to use a common method in `@platformatic/utils` which will always use the right arguments and do `.catch()`.